### PR TITLE
[EDU-3958] limits edge functions

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: [EDU-3958](https://aziontech.atlassian.net/browse/EDU-3958)
Changes
Added Limits to Edge Functions reference in EN and PT.

[EDU-3958]: https://aziontech.atlassian.net/browse/EDU-3958?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-functions.mdx [hunk 1]
```diff
@@ -187,6 +187,7 @@ Learn more about the management of edge functions trough [the Azion Terraform Pr
 | Memory per Isolate | 512mb |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-functions.mdx:187]: Em where estamos deixando feature/funcionamento

Comment (gabriel-azion) [reply src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-functions.mdx:187]: feature/funcionalidade ?

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-functions.mdx [hunk 2]
```diff
@@ -187,6 +187,7 @@ Learn more about the management of edge functions trough [the Azion Terraform Pr
...
 | Code Limit (UI) | 6mb |
 | Code Limit (API) | 20MB |
+| Environment variable size | 32kb |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-functions.mdx:190]:
```suggestion
| Memory per Isolate | 512 mb |
| Code Limit (UI) | 6 mb |
| Code Limit (API) | 20 MB |
| Environment variable size | 32 kb |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-functions-edge-app.mdx [hunk 3]
```diff
@@ -188,6 +188,7 @@ Saiba mais sobre o gerenciamento de edge functions através do [Azion Terraform
 | Memória por Isolate | 512mb |
 | Limite de código (UI) | 6mb |
 | Limite de código (API) | 20MB |
+| Tamanho da variável de ambiente | 32kb |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-functions-edge-app.mdx:191]:
```suggestion
| Memória por Isolate | 512 mb |
| Limite de código (UI) | 6 mb |
| Limite de código (API) | 20 MB |
| Tamanho da variável de ambiente | 32 kb |
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-functions.mdx (hunk 2 above)
Comment (MarianaAguilera) [reply src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-functions.mdx:190]: ajustar primeiro item e alinhar units

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-functions-edge-app.mdx (hunk 3 above)
Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-functions-edge-app.mdx:191]: idem

**Pull request merged**